scilpy/tractanalysis/features.py

```diff
@@ -10,6 +10,45 @@
 import numpy as np
 
 
+def detect_ushape(streamlines, ufactor=[0.5, 1]):
```

Comment:
In the function I would give a detailled example of what u factor means.
Like in the fibercub there is a u bundle, could you extract it with this function?
The AF is a BIG u shape, can it be detected?

(Also the whole docstrinc is a copy paste and there is a bla bla bla)